bibliopixel/animation/animation.py

```diff
@@ -30,19 +31,18 @@ def _led(self):
     def _led(self, layout):
         self.layout = layout
 
-    def preRun(self, amt=1):
-        self.layout.all_off()
-
     @property
-    def complete_callback(self):
-        return self.__complete_callback
+    def completed(self):
+        """Many BiblioPixelAnimations use the old `completed` variable."""
+        return self.state == State.Complete
 
-    @complete_callback.setter
-    def complete_callback(self, callback):
-        if not callable(callback):
-            raise ValueError('complete_callback must be a function!')
+    @completed.setter
+    def completed(self, state):
+        if state:
+            self.state = State.Complete
```

Comment:
Yeah... fair enough. It's one of those things that I ever think of as starting out False and getting set to True when the animation is done. I've updated the logic.